src/components/main/group/focused-word/FocusedWord.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../../style/theme.scss';
+
+.focused-word {
+  &__spinner {
+    @include spinner(8px, $primary-color, 1s);
+    height: 100px;
+    width: 100px;
+    margin: 2rem auto;
+  }
+
+  &__owlbot-info {
+    display: grid;
+    gap: 1rem;
+  }
+  
+  &__owlbot-definition-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
+  }
+
+  &__owlbot-definition-container {
+    display: grid;
```

Comment:
can group the display: grid & gap: 1rem together for this and &__owlbot-definition-list